### DISCUSSION TITLE
Welcome to CRBT Discussions!

### DISCUSSION THREAD

**Discussion by clembs** [CRBT-Team/CRBT#10] [Announcements]
Welcome to the Discussions page for CRBT. Here you can discuss CRBT, ask for help, suggest new features and engage with the community.

### Helpful resources

* [Discord server](https://crbt.app/discord) - Extra news, support, reports, etc.
* [Issues](https://github.com/CRBT-Team/CRBT/issues) - To report bugs, typos, and unexpected behavior
* [Community Guidelines](https://docs.github.com/articles/github-community-guidelines) - You *can* read them before posting here, but common sense always works

**Comment by SonicQ45** [comment]
👀
